pkg/crd/gen.go

```diff
@@ -76,16 +89,45 @@ func (g Generator) Generate(ctx *genall.GenerationContext) error {
 		return nil
 	}
 
+	crdVersions := g.CRDVersions
+
+	if len(crdVersions) == 0 {
+		// put v1beta1 first to make it the "default" file for compatibility purposes
+		crdVersions = []string{"v1beta1", "v1"}
```

Comment:
Not sure if this requires major version bump for controller-tools. In a way, it changes the default behavior. I am fine with bumping up `0.3.0`.

Reply:
I don't think it's a problem because v1beta1 is still in the same named file.

Reply:
we can just not generate v1 at all by default as well

Reply:
let's just do that and eventually change to make this the default